src/test/FuzzerImpl.cpp

```diff
@@ -70,14 +70,14 @@ getShortKey(PublicKey const& pk)
 }
 
 // constructs an Asset structure for an asset issued by an account id comprised
-// of bytes reading [0,0,...,i] and an alphanum4 asset code of "Ast + i"
+// of bytes reading [0,0,...,digit] and an alphanum4 asset code of "Ast + digit"
```

Comment:
I've tried to make the comment clearer.  I left the `digit` change because I still like it better than `i` in this case, but if you prefer `i` then I am fine with going back to it.